.circleci/config.yml

```diff
@@ -398,6 +450,7 @@ workflows:
             - api-deps-check
             - api-integration-test
```

Comment:
Could you add the puppeteer tests to the staging deploy job as well? We should probably block releases on this?

Reply:
> Could you add the puppeteer tests to the staging deploy job as well? We should probably block releases on this?

This sounds like the right approach on the surface (at least until we can run these tests hermetically pre-merge, which would provide a strong guarantee that "any merged code passes our e2e tests"). We should probably let this burn in for awhile before potentially disrupting our release process. This sounds like a good point to discuss in the design doc (@aweng98 can you add a link to the design from RW-4139?) and potentially discuss at tomorrow's eng sync.

Reply:
My recommendation is take some time to learn more about tests, test framework and Puppeteer. Let's keep it simple and non-disruptive. Only enable this job for test-deploying and non-blocking deploy and releases.